indico/MaKaC/webinterface/tpls/PosterTemplates.tpl

```diff
@@ -14,7 +14,7 @@
     % for template in templateList:
     <li>
         <span class="name">${template['name']}</span>
-        <div class="toolbar right thin inline">
+        <div class="right thin inline">
```

Comment:
This change results in the buttons showing up separately instead of tied together...